mypy failures with mypy 2.2.0

`mypy` 2.2.0 came out about 2 days ago and we are having type checking failures with it:

```sh
🚀 Static type checking: Running mypy
cmd2/annotated.py:300: note: In module imported here,
examples/getting_started.py:40: note: ... from here:
cmd2/argparse_utils.py: note: In member "_get_formatter" of class "Cmd2ArgumentParser":
cmd2/argparse_utils.py:1038:5: error: Signature of "_get_formatter" incompatible with supertype "argparse.ArgumentParser"  [override]
cmd2/argparse_utils.py:1038:5: note:      Superclass:
cmd2/argparse_utils.py:1038:5: note:          def _get_formatter(self, file: SupportsWrite[str] | None = ...) -> HelpFormatter
cmd2/argparse_utils.py:1038:5: note:      Subclass:
cmd2/argparse_utils.py:1038:5: note:          def _get_formatter(self, **_kwargs: Any) -> Cmd2HelpFormatter
cmd2/argparse_utils.py: note: In member "format_help" of class "Cmd2ArgumentParser":
cmd2/argparse_utils.py:1042:5: error: Signature of "format_help" incompatible with supertype "argparse.ArgumentParser"  [override]
cmd2/argparse_utils.py:1042:5: note:      Superclass:
cmd2/argparse_utils.py:1042:5: note:          def format_help(self, formatter: HelpFormatter | None = ...) -> str
cmd2/argparse_utils.py:1042:5: note:      Subclass:
cmd2/argparse_utils.py:1042:5: note:          def format_help(self) -> str
Found 2 errors in 1 file (checked 24 source files)
```

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -1,3 +1,8 @@
+## 4.1.1 (TBD)
+
+- Bug Fixes
+    - Fixed a couple type issues that emerged with the upgrade to `mypy` 2.2.0
+
 ## 4.1.0 (July 7, 2026)
 
 - Breaking Changes

--- a/cmd2/argparse_utils.py
+++ b/cmd2/argparse_utils.py
@@ -1035,13 +1035,13 @@ class Cmd2ArgumentParser(argparse.ArgumentParser):
 
         self.exit(2, f"{formatted_message}\n")
 
-    def _get_formatter(self, **_kwargs: Any) -> Cmd2HelpFormatter:
+    def _get_formatter(self, *_args: Any, **_kwargs: Any) -> Cmd2HelpFormatter:
         """Override with customizations for Cmd2HelpFormatter."""
         return self.formatter_class(prog=self.prog, file=self._thread_locals.current_output_file)
 
-    def format_help(self) -> str:
+    def format_help(self, *args: Any, **kwargs: Any) -> str:
         """Override to add a newline."""
-        return super().format_help() + "\n"
+        return super().format_help(*args, **kwargs) + "\n"
 
     def _get_nargs_pattern(self, action: argparse.Action) -> str:
         """Override to support nargs ranges."""

